apps/client/tailwind.config.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
       text: {
         light: '#F4DDDB',
       },
+      bg:{
+        brown: '#3B2D2C',
+        'brown-100': '#6B5A59',
```

Comment:
Do we have other brown variants in the design? If so shall we include them all

Reply:
These were changed in PR #43 under commit https://github.com/the-ai-team/razor/pull/43/commits/c50411361f07279c7eda2da51c2712b1b41c052a